app/screens/settings-screen/language-screen.tsx

```diff
@@ -43,7 +42,7 @@ export const LanguageScreen: ScreenType = () => {
                     errors: [],
                     user: {
                       __typename: "User",
-                      id: tokenUid,
+                      id: me?.id,
```

Comment:
fyi (I don't think there is anything to change):
- me.id will be returning the Account.id 
- user id will eventually be be migrated to be the kratosUserId